lib/change_stream.js

```diff
@@ -483,11 +475,8 @@ function processNewChange(args) {
     if (eventEmitter) {
       return;
     }
-
-    const error = new MongoError('ChangeStream is closed');
-    return typeof callback === 'function'
-      ? callback(error, null)
-      : changeStream.promiseLibrary.reject(error);
+    callback(new MongoError('ChangeStream is closed.'));
```

Comment:
nit: no period at the end of the message (acknowledging that you didn't introduce this)